packages/customize-widgets/src/components/sidebar-block-editor/index.js

```diff
@@ -118,7 +124,12 @@ export default function SidebarBlockEditor( {
 						<WritingFlow>
 							<ObserveTyping>
 								<BlockList
-									renderAppender={ ButtonBlockAppender }
+									renderAppender={ ( props ) => (
```

Comment:
There's a flaw with `renderAppender` that it doesn't work too well as a function prop (essentially it's like giving React a new component on every render), so it'd be good to `useCallback` on the function to avoid that.

Reply:
An alternative could be to move the effect to ButtonBlockAppender to avoid that.